app/Http/Controllers/Api/PostController.php

```diff
@@ -99,4 +99,9 @@ public function postsByFilters(Request $request)
 
     return response()->json($posts);
   }
+
+  public function postsFeatured()
+  {
+    return Post::where('featured', 1)->limit(5);
```

Comment:
Il return e la query devono essere su due righe diverse. Manca il response in json